src/popup/App.vue

```diff
@@ -127,11 +128,13 @@ export default defineComponent({
     const { modalsOpen } = useModals();
     const { isLoggedIn } = useAccounts();
     const { addWalletNotification } = useNotifications();
-    const { loadCoinsData } = useCurrencies({ withoutPolling: true });
+    const { loadCoinsData } = useCurrencies({ pollingDisabled: true });
     const { restoreLanguage } = useLanguages();
     const { restore: restoreTransferSendForm } = useTransferSendHandler();
     const { multisigAccounts } = useMultisigAccounts({ pollingDisabled: true });
 
+    usePendingMultisigTransaction(); // Ensure the persistency of the watchers
```

Comment:
Is this actually the only way to actually remain watching? Because this composable is used within the `PendingMultisigTransactionCard` already, and it is presented in `/account`.

Reply:
This is the only solution I came up. The main reason is that the `watch` function behaves differently based on the composable's scope
1. If the composable is fired by any **JS/TS file** (including another composable) the watcher is persistent. So for every JS/TS file new instance of the watcher is created. If we don't block running multiple watchers the impact on app's performance is quite big.
2. If the composable is fired within a Vue component the watcher is bind to the component so if the component is unmounted the watcher is also destroyed. 

These two possibilities makes the app slightly unpredictable because depending on what kind of file fires the composable for the first time we can't be sure what will happen with the watcher. For most of the composables the 1st option is applicable so the `if (!composableInitialized)` makes 100% sense. The `pendingMultisigTransaction` is fired for the 1st time by the Vue component in the case you described @CedrikNikita so it's removed when we leave the page :)

The line we are discussing right now makes sure that the composable is first run in scope that would not remove the watcher.

If I will remove the `if (!composableInitialized)` from the composable the watcher is fired multiple times and causes multiple API calls.

If you think there is a better solution please share.

Reply:
This only means that there could be more places like that. Shall we now move all composables with global watchers (that can be initialized on vue scope) to this file?

Reply:
I think this would be rather bad to just initiate all composables with watchers in the App.vue file.
I came up with different approach to the problem. Please see this comment: https://github.com/superhero-com/superhero-wallet/pull/2776/files#r1514116021 and the code related to it.